openmls/src/framing/public_message.rs

```diff
@@ -112,17 +117,23 @@ impl PublicMessage {
 impl From<AuthenticatedContent> for PublicMessage {
     fn from(v: AuthenticatedContent) -> Self {
         Self {
-            content: v.tbs.content,
+            content: v.tbs,
             auth: v.auth,
             membership_tag: None,
         }
     }
 }
 
+impl From<PublicMessage> for VerifiableAuthenticatedContent {
```

Comment:
Should this be possible? Isn't the `VerifiableAuthenticatedContent` there to enforce verification?

Reply:
Yes, but it's downstream of PublicMessage. After validating the membership tag, we have to turn the PublicMessage into a VerifiableAuthenticatedContent. Ideally, there would be distinct types for before and after MembershipTag validation, but I think that's something we want to do generically for MAC validation at some point.